README.md

```diff
@@ -54,6 +54,17 @@ Installation in Composer project (method 2)
 * Use the coding standard with `./vendor/bin/phpcs --standard=PHPCompatibility`
 
 
+Installation via a git check-out to an arbitrary directory (method 3)
+-----------------------
+
+* Install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) via [your preferred method](https://github.com/squizlabs/PHP_CodeSniffer#installation) (Composer, PEAR, Phar file, Git checkout).
+* Checkout the latest release from https://github.com/wimg/PHPCompatibility/releases into an arbitrary directory.
+* Add the path to the directory **above** the directory in which you cloned the PHPCompability repo to the PHPCS Configuration:
```

Comment:
Not sure if this sentence is clear, at least to me it isn't. It sounds like you need to edit a configuration and add a path to the line before an existing line.